src/lesson4/task1/List.kt

```diff
@@ -104,14 +105,26 @@ fun buildSumExample(list: List<Int>) = list.joinToString(separator = " + ", post
  * по формуле abs = sqrt(a1^2 + a2^2 + ... + aN^2).
  * Модуль пустого вектора считать равным 0.0.
  */
-fun abs(v: List<Double>): Double = TODO()
+fun abs(v: List<Double>): Double {
+    var length = 0.0
+    for (i in 0..v.size - 1) {
+        length = length + v[i] * v[i]
```

Comment:
Здесь лучше применить перебор по элементам: `for (elem in v) { /* use elem instead of v[i] */ }`. Лучше не делать индексацию там, где можно без этого обойтись - не у любого списка она выполняется быстро